nbbuild/javadoctools/export2apichanges.xsl

```diff
@@ -101,7 +101,9 @@
                 <a href="http://bits.netbeans.org/7.4/javadoc/apichanges.html">release 7.4</a>,
                 <a href="http://bits.netbeans.org/8.0/javadoc/apichanges.html">release 8.0</a>,
                 <a href="http://bits.netbeans.org/8.0.1/javadoc/apichanges.html">release 8.0.1</a>,
-                <a href="http://bits.netbeans.org/8.1/javadoc/apichanges.html">release 8.1</a>.
+		<a href="http://bits.netbeans.org/8.1/javadoc/apichanges.html">release 8.1</a>,
+		<a href="http://bits.netbeans.org/8.2/javadoc/apichanges.html">release 8.2</a>,
+		<a href="http://bits.netbeans.org/9.0/javadoc/apichanges.html">release 9.0</a>.
```

Comment:
Dead link

Reply:
apidoc for Apache 9.0 is not released yet it's need branding too see PR #1022. 
8.1 and 8.2 links are ok in my place.

Reply:
Great, thanks !